### ISSUE TITLE
How can I evaluate model with pascal voc2007?

### ISSUE THREAD

**Issue by ksw9446** [fizyr/keras-retinanet#218]
I want to get mAP of trained model in voc2007 test set.
please help me..

**Comment by de-vri-es** [comment]
You should be able to measure the performance metrics you are interested in. Is there a specific problem with `keras-retinanet` you are having which prevents you from measuring the performance?

**Comment by ksw9446** [comment]
sorry. I didn't know 'eval.py'.
but now, I got mAP using 'eval.py'
